lib/src/tooltip_widget.dart

```diff
@@ -439,44 +443,52 @@ class _ToolTipWidgetState extends State<ToolTipWidget>
                                     : CrossAxisAlignment.center,
```

Comment:
Please see if this is still required after the usage of `Align` as parent of the children.